packages/blade/src/components/Tag/__tests__/Tag.web.test.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import userEvents from '@testing-library/user-event';
+import React from 'react';
+import { Tag } from '../Tag';
+import renderWithTheme from '~utils/testing/renderWithTheme.web';
+
+describe('<Tag />', () => {
+  it('should render tag', () => {
+    const { container } = renderWithTheme(
+      <Tag
+        onDismiss={() => {
+          console.log('tag dismissed');
+        }}
+      >
+        in:User
+      </Tag>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should call onDismiss', async () => {
+    const user = userEvents.setup();
+    const dismissHandler = jest.fn();
+    const { getByRole, queryAllByText } = renderWithTheme(
+      <Tag onDismiss={dismissHandler}>in:User</Tag>,
+    );
+
+    const [_desktopTagText, mobileTagText] = queryAllByText('in:User');
```

Comment:
what is desktop tag and mobile tag? 🤔

Reply:
There will tags of 2 sizes in DOM and we switch between them with media query (explained it [here](https://github.com/razorpay/blade/pull/1361#discussion_r1275745922))